Backend/src/utils/filesystem.py

```diff
@@ -36,3 +40,28 @@ def read_meta_info(file_name: str) -> dict:
             raise ValueError(f"Error parsing JSON from file '{meta_info_path}': {e}")
 
     return meta_info_data
+
+def download_qaqc_source_dataset() -> pd.DataFrame:
```

Comment:
The function `download_qaqc_source_dataset` should maybe be placed somewhere else. From what I see it does not use the filesystem. I suggest creating a `test_helper.py` or equivalent in the `utils` folder.

Reply:
done